Create a call for reviewers pull request template (#566)
fortran-lang/stdlib: .github/PULL_REQUEST_TEMPLATE.md

<!--
Thank you for contributing to stdlib.

> To help us get your pull request merged more quickly, you can help us with reviewing any of the already open pull requests.

Reviewer: I suggest more active wording

```suggestion
To help us get your pull request merged more quickly, please consider reviewing any of the already open pull requests.
```
